assets/js/components/SaptuneDetails/SaptuneDetailsPage.test.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import {
+  renderWithRouterMatch,
+  withState,
+  defaultInitialState,
+} from '@lib/test-utils';
+import { hostFactory } from '@lib/test-utils/factories';
+
+import SaptuneDetailsPage from './SaptuneDetailsPage';
+
+describe('SaptuneDetailsPage', () => {
+  it('should render not found as the host is miissing', () => {
```

Comment:
I'd use `when` instead of `as`. Also, miising => missing

Reply:
ah thanks for the nice catch!